### PR TITLE
3 New Enchants, 13 Rebuilt

### PR DESCRIPTION
**Pull request by btjt97**

### CODE REVIEW

File: eco-core/core-plugin/src/main/resources/enchants/smelting.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+tradeable: true 
+discoverable: true
+enchantable: true 
+
+effects:
+  - id: autosmelt
+    args:
+      drop_xp: true
+
```
Comment (MillionthOdin16) [inline eco-core/core-plugin/src/main/resources/enchants/smelting.yml:23]: Pretty sure you need a item drop event trigger

effects:
  - id: autosmelt
    args:
      drop_xp: true
    triggers:
      - block_item_drop

**Review by MillionthOdin16 — CHANGES_REQUESTED** [review]
Tweak smelting effect

**Comment by MillionthOdin16** [comment]
Did you even test these? Loading these enchants in the beta clearly shows configuration errors. Mostly doing with missing triggers.
For example:

`[EcoEnchants] Invalid configuration for damage_multiplier in context Enchantment endinfusion:  
[EcoEnchants] (Cause) Argument 'triggers'  
[EcoEnchants] (Reason) Specified effect requires at least 1 trigger`

**Comment by WillFP** [comment]
You didn't test these before PRing?

**Comment by MillionthOdin16** [comment]
@btjt97 I submitted a PR to you that has updates with fixes. Stab enchant looked pretty misconfigured, so I removed it.

**Pull request closed without merging**